dist/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Webpack Exercise</title>
+    <!-- <link rel="stylesheet" href="style.css">
+    <script src="./index.js" defer></script> -->
+<script defer src="main.js"></script></head>
+<body>
+
+<section class="todo-list">
+    <div id="myDIV" class="header">
+        <div class="header-container flex-between">
+        <h3>My To Do List</h3>
+        <img class="w4"src="../assets/refresh-svgrepo-com.svg" alt="refresh">
+        </div>
+        
+        <hr>
+       
+        
+        <input type="text" id="myInput" placeholder="Title...">
+      
+        <hr>
+    </div>
+
+    <ul class="items" id="items">
+        
+        <!-- <li class="flex-between">
+
+            <div class="container-input">
+            <input id="item1" type="checkbox" class="checkbox"checked>
+            <label for="item1">Create a to-do list</label>
+            
+            </div>
+                <img class="w4"src="../assets/three-dots-svgrepo-com.svg" alt="">
+    
+        </li>
+        <hr>
+        <li class="flex-between"> -->
+
+            <!-- <div class="container-input">
+            <input id="item1" type="checkbox" class="checkbox"checked>
+            <label for="item1">Create a to-do list</label>
+            
+            </div>
+                <img class="w4"src="../assets/three-dots-svgrepo-com.svg" alt="">
+    
+        </li>
```

Comment:
Kindly do not forget to remove comments from your previous code,  It's important to remove comments from code before sharing it with others because comments can make the code harder to read and understand.